CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Version 1.15.0/0.36b0 (2022-12-10)
 
+- Add uninstrument test for celery
+  ([#1490](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1490))
```

Comment:
This version is already released. And we don't need to add an entry here for this because it's not a concern for the end user.

Reply:
Removed, please add me a "Skip Changelog" label 🙏